### PR TITLE
Get rid of usage rispa-vendor plugin

### PR DESCRIPTION
**Pull request by wuzyk**
Use webpack aliases to solve problem with react-loadable

### CODE REVIEW

**Comment by Laiff** [comment]
What kind of problems you found?

**Comment by wuzyk** [comment]
`react-loadable` implementation assumes that there is the only one instance of `react-loadable` in the project. `vendor` plugin used for providing `react-loadable` for plugins and solves issue with multiple instances.
But I found another solution. It is to use webpack alias for `react-loadable`. So `vendor` plugin is not needed any more

**Review by Laiff — DISMISSED** [review]
Nice catch!

**Comment by Laiff** [comment]
Now we need to make some changes in other plugins which depends on `rispa-vendor` plugin.
Moved to separete task

**Comment by wuzyk** [comment]
[46 Rid of rispa-vendor plugin](https://trello.com/c/lQjtaNyA/46-46-rid-of-rispa-vendor-plugin)

File: .rispa/babel-options.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default () => ({
+  plugins: [
+    [
+      require.resolve('react-loadable/babel'),
+      { server: false, webpack: true },
```
Comment (Laiff) [inline .rispa/babel-options.js:5]: The plugin for the babel now works correctly because webpack aliases are used?

**Review by Laiff — APPROVED** [review]

File: .rispa/babel-options.js (hunk 1 above)
Comment (wuzyk) [reply .rispa/babel-options.js:5]: right

**Pull request merged**